### PR TITLE
refactor folder structure in blocks directory

### PR DESCRIPTION
**Pull request by chetanyagoyal**
The new `blocks` folder looks like - 
- blocks 
  - elementary
    - diff_pair
      - ..
    - current_mirror
      - ..
  - composite
    - opamp
      - ..
    - ..

### CODE REVIEW

**Comment by chetanyagoyal** [comment]
@msaligane this can be merged

**Review by alibillalhammoud — CHANGES_REQUESTED** [review]
You correctly updated all dependencies and paths in the syntaxer folder. strict syntax -> python conversion still works. Let us hold off on merging until #333  is merged

**Review by harshkhandeparkar — COMMENTED** [review]
Looks good to me.

**Review by msaligane — APPROVED** [review]

**Pull request merged**
